doc/js-api/remotestorage.rst

```diff
@@ -16,6 +16,43 @@ example::
      cordovaRedirectUri: 'https://app.wow-much-app.com' // defaults to undefined
    });
 
+Here is a complete list of the default options (set in ``src/config/js``), all
+of which can be overridden by calling the constructor with a configuration
+object like above::
+
+   {
+     // Enable remoteStorage logging
+     logging: false,
+     // Change Events that are enabled. See the BaseClient API for more information
+     changeEvents: {
+       local:    true,
+       window:   false,
+       remote:   true,
+       conflict: true
+     },
+     // enable caching
+     cache: true,
+     // disable specified modules
+     disableFeatures: [],
+     // timeout for the Webfinger lookup, discovering a connecting user's storage details
+     discoveryTimeout: 10000,
```

Comment:
Not public.

Reply:
I thought it was because of this comment: https://github.com/remotestorage/remotestorage.js/blob/b01fa56028e5bc18e4f7ef1782c045a77ccb3dc3/src/discover.js#L20-L21

Reply:
I don't how that got there, and it's not part of any public docs, but why would one want to change this value?